openml/runs/trace.py

```diff
@@ -1,6 +1,10 @@
 import arff
 import json
 import os
+import xmltodict
+from collections import OrderedDict
+
+PREFIX = 'parameter_'
```

Comment:
Personally I don't like global variables. I don't remember the exact meaning of global variables in Python, is this variable now usable across all files or only in this file? In case of the prior, can this become a static class variable?

Reply:
This is not a global variable in the sense that it can be called from everywhere, it can only be called inside the module. Also, I'm not aware that there is something like a static class variable in python, so I think this is the best and most common solution to declare such a constant.

Reply:
ok